src/containers/widget/index.js

```diff
@@ -22,24 +22,31 @@ import { WidgetWrapper } from "../../utils/widget-wrapper";
 // Merge various configuration objects. The last has the highest priority.
 // usage: getInitialState(this.props, {title: "Hallo"});
 function getInitialState() {
-  return Object.assign(Widget.defaultProps, ...arguments);
+  const initialState = Object.assign(Widget.defaultProps, ...arguments);
+  return {
+    ...initialState,
+    isModalOpen: false,
+    invoiceDetails: null,
+    fetchingInvoiceState: false,
```

Comment:
Can we add all these three (`isModalOpen`, `invoiceDetails` & `fetchingInvoiceState`) to `Widget.defaultProps`? I don't see a point in adding them here explicitly.

Reply:
sure, that's just how it was currently. but yeah the goal was to simplify all that and have those settings in one place. 
:+1: